### PR TITLE
Doc on Delegated Administration extension

### PR DESCRIPTION
**Pull request by mpaktiti**
Documentation for our new extension: Delegated Administration.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mpaktiti** [comment]
[Preview app](https://auth0-docs-content-pr-2047.herokuapp.com/docs/extensions/delegated-admin)

File: articles/extensions/delegated-admin.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    </tr>
+</table>
+
+Notice the new *Reset password* option available via the extension. This option will send an email to the user allowing them to choose a new password. To do this click on a user and select *Actions > Reset password*.
+
+![](/media/articles/extensions/delegated-admin/reset-pass-01.png)
+
+This will send an email to the user, containing a link to change the password.
+
+At the *Logs* view you can see log data of authentications made by your users. The contents of this view are a subset of the data displayed in the [Logs dashboard](${uiURL}/#/logs), which also displays data for the actions taken in the dashboard by the administrators.
```
Comment (mpaktiti) [inline articles/extensions/delegated-admin.md:139]: @sandrinodimattia Is this section on the Logs view valid? I can see the `API Operation` logs only on my dashboard, they are not displayed via the extension.

File: articles/extensions/delegated-admin.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+Let's start with creating a new client application. Navigate to [Clients](${uiURL}/#/applications) and click on the **+Create Client** button. Set a name (we will name ours *Users Dashboard*) and choose *Single Page Web Applications* as client type. Click on **Create**.
+
+![](/media/articles/extensions/delegated-admin/create-client.png)
+
+Navigate to *Settings > Show Advanced Settings > OAuth* and set the **JsonWebToken Signature Algorithm** to *RS256*. Click **Save Changes**.
+
+![](/media/articles/extensions/delegated-admin/set-rs256.png)
+
+While on the *Settings* tab set the **Allowed Callback URLs**. This varies based on your location.
```
Comment (mpaktiti) [inline articles/extensions/delegated-admin.md]: @sandrinodimattia At the moment there is not variable I can use so I listed the options. I registered [this issue](https://github.com/auth0/auth0-docs/issues/994). When resolved I will update the doc.

**Pull request merged**